app/models/manageiq/providers/ibm_cloud/inventory/parser/power_virtual_servers.rb

```diff
@@ -290,9 +290,10 @@ def cloud_volume_types
     # get only the active storage
     collector.storage_types.storage_types_capacity.each do |v|
       persister.cloud_volume_types.build(
-        :type    => "ManageIQ::Providers::IbmCloud::PowerVirtualServers::StorageManager::CloudVolumeType",
-        :ems_ref => v.storage_type,
-        :name    => v.storage_type
+        :type        => "ManageIQ::Providers::IbmCloud::PowerVirtualServers::StorageManager::CloudVolumeType",
+        :ems_ref     => v.storage_type,
+        :name        => v.storage_type,
+        :description => v.storage_type
```

Comment:
Do you mind adding the `:description` attribute to our [assert_specific_cloud_volume spec](https://github.com/ManageIQ/manageiq-providers-ibm_cloud/blob/master/spec/models/manageiq/providers/ibm_cloud/power_virtual_servers/cloud_manager/refresher_spec.rb#L234-L239) since the create cloud volume form doesn't work without it?

Reply:
Sure thing, addressed in the latest force-push.